lessons/pydata/pandas_types/index.ipynb

```diff
@@ -0,0 +1,6941 @@
+{
```

Comment:
<p>Já se čím dál víc přikláním k <code>assign</code>. Kromě obecných problémů mutací si to lépe poradí s kopiemi, které vznikly nějakým transformacemi apod.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Něco jsem tam zkusil napsat.</p>

Reply:
👍